### PR TITLE
JsonConverter reader implementation

### PR DESCRIPTION
**Pull request by mateuszanini**
Reader implementation in JsonConverter for DictionaryToSemicolonSplittedStringConverter and NumericFlagConverter.
Useful for serialization and deserialization process when data is persisted in a database, for example.

### CODE REVIEW

**Comment by george-tsaplin** [comment]
@mateuszanini , thanks for suggestion. Can you add some unit-tests on new code?

**Comment by mateuszanini** [comment]
> @mateuszanini , thanks for suggestion. Can you add some unit-tests on new serialization code?

Done!

File: CapMonsterCloud.Client.Tests/JsonConverterTests.cs [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                Body = "some base64 body",
+                CapMonsterModule = CapMonsterModules.YandexWave,
+                CaseSensitive = true,
+                Numeric = numeric,
+                RecognizingThreshold = 65,
+                Math = false
+            };
+
+            // Act
+            var actual = JsonConvert.SerializeObject(target);
```
Comment (george-tsaplin) [inline CapMonsterCloud.Client.Tests/JsonConverterTests.cs:27]: Act section should contain only one operation, which is under test. In these tests I suppose it should be `JsonConvert.DeserializeObject...`

**Review by george-tsaplin — CHANGES_REQUESTED** [review]
I beleave it will be last change request and we can merge it

File: CapMonsterCloud.Client.Tests/JsonConverterTests.cs (hunk 1 above)
Comment (mateuszanini) [reply CapMonsterCloud.Client.Tests/JsonConverterTests.cs]: I made changes in JsonConverterTests methods to match the AAA pattern, as you asked.

**Review by george-tsaplin — APPROVED** [review]

**Pull request merged**